stdlib/contrib/qxip/logql/logql.flux

```diff
@@ -0,0 +1,87 @@
+// Package logql provides functions meant to simplify common logql queries.
```

Comment:
Our project [qryn](https://qryn.dev) is also in scope and this integration attempts to be neutral, hence the generic LogQL definition rather than limiting this to Loki. We have not included our own link for the same reason. 
This said, If you think this would be best for the documentation, we have absolutely no objections.